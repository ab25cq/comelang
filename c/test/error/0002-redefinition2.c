/*
PATTERN: L1
PATTERN: c:11
*/

void
foo()
{
	L1:
	L2:
	L1:
	return;
}
