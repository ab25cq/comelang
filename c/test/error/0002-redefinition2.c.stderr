redefinition of label L1 at test/error/0002-redefinition2.c:11:5
    L1:
    ^
